<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.0.3 (Python 3.9.5 on win32)" generated="20210706 20:50:30.301" rpa="false" schemaversion="2">
<suite id="s1" name="Mobile Test" source="c:\Users\gabri\OneDrive\Documentos\PrimeExperts Projects\PrimeExperts_junho_28_2021\mobile_test\mobile_test.robot">
<kw name="Abrir o aplicativo no celular" library="resource_mobile" type="SETUP">
<kw name="Open Application">
<arg>${REMOTE_URL}</arg>
<arg>platformName=${PLATFORM_NAME}</arg>
<arg>platformVersion=${PLATFORM_VERSION}</arg>
<arg>deviceName=${DEVICE_NAME}</arg>
<arg>appPackage=${APP_PACKAGE}</arg>
<arg>appActivity=${APP_ACTIVITY}</arg>
<arg>automationName=${AUTOMATION_NAME}</arg>
<msg timestamp="20210706 20:50:30.710" level="FAIL">No keyword with name 'Open Application' found.</msg>
<status status="FAIL" starttime="20210706 20:50:30.710" endtime="20210706 20:50:30.710"/>
</kw>
<status status="FAIL" starttime="20210706 20:50:30.703" endtime="20210706 20:50:30.710"/>
</kw>
<test id="s1-t1" name="Pesquisar canal &quot;Adrenaline&quot;">
<status status="FAIL" starttime="20210706 20:50:30.711" endtime="20210706 20:50:30.712">Parent suite setup failed:
No keyword with name 'Open Application' found.</status>
</test>
<doc>arquivo responsavel pelos testes do mobile testing App: YouTube</doc>
<status status="FAIL" starttime="20210706 20:50:30.324" endtime="20210706 20:50:30.713">Suite setup failed:
No keyword with name 'Open Application' found.</status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Mobile Test">Mobile Test</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20210706 20:50:30.695" level="ERROR">Error in file 'c:\Users\gabri\OneDrive\Documentos\PrimeExperts Projects\PrimeExperts_junho_28_2021\mobile_test\resource_mobile.robot' on line 3: Importing library 'AppiumLibrary' failed: ModuleNotFoundError: No module named 'AppiumLibrary'
Traceback (most recent call last):
  None
PYTHONPATH:
  c:\Users\gabri\.vscode\extensions\robocorp.robotframework-lsp-0.18.0\src\robotframework_debug_adapter
  c:\Users\gabri\OneDrive\Documentos\PrimeExperts Projects\PrimeExperts_junho_28_2021
  c:\Users\gabri\.vscode\extensions\robocorp.robotframework-lsp-0.18.0\src
  c:\Users\gabri\.vscode\extensions\robocorp.robotframework-lsp-0.18.0\src\robotframework_ls\vendored
  C:\Users\gabri\AppData\Local\Programs\Python\Python39\python39.zip
  C:\Users\gabri\AppData\Local\Programs\Python\Python39\DLLs
  C:\Users\gabri\AppData\Local\Programs\Python\Python39\lib
  C:\Users\gabri\AppData\Local\Programs\Python\Python39
  C:\Users\gabri\AppData\Local\Programs\Python\Python39\lib\site-packages</msg>
</errors>
</robot>
